src/awkward/_v2/index.py

```diff
@@ -1,79 +1,84 @@
+# BSD 3-Clause License; see https://github.com/scikit-hep/awkward-1.0/blob/main/LICENSE
+
+from __future__ import absolute_import
+
 import awkward as ak
 
+np = ak.nplike.NumpyMetadata.instance()
```

Comment:
Although functions are split among `nplike` singletons, some basic types like `np.int8` and `np.dtype` are in a metadata singleton, which we call `np` everywhere (as opposed to the full "numpy", "cupy", or "nplike").